src/exception/exception-api.ts

```diff
@@ -77,15 +65,20 @@ class ExceptionController {
     }
   }
 
+  @HasFullWriteAccess()
+  async clearProgramException(req: Request, res: Response) {
+    const programId = req.params.programId;
+    const result = await exceptionService.operations.deleteProgramException({ programId });
```

Comment:
While I like the DRY approach you went for, the `try/catch` in the original implementation ensures catching any other possible errors within `...deleteProgramException`.
perhaps you can wrap this in it again, and add respond with a 500 if there's any issues?